test/models/item_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
+
+class ItemTest < ActiveSupport::TestCase
+  attr_reader :item
+
+  def setup
+    @item = Item.create(name: 'espresso')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.